extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtension.scala

```diff
@@ -43,9 +43,11 @@ class RangerSparkExtension extends (SparkSessionExtensions => Unit) {
     v1.injectResolutionRule(_ => new RuleReplaceShowObjectCommands())
     v1.injectResolutionRule(_ => new RuleApplyPermanentViewMarker())
     v1.injectResolutionRule(new RuleApplyRowFilterAndDataMasking(_))
+    v1.injectResolutionRule(new RuleSetTableOwner(_))
     v1.injectOptimizerRule(_ => new RuleEliminateMarker())
     v1.injectOptimizerRule(new RuleAuthorization(_))
     v1.injectOptimizerRule(_ => new RuleEliminateViewMarker())
     v1.injectPlannerStrategy(new FilterDataSourceV2Strategy(_))
+    v1.injectPlannerStrategy(new SeTableOwnerStrategy(_))
```

Comment:
I don't know whether good enough to introduce a pair of one resolution rule plus one plan strategy here, specialized for table setting senairos.

Is there any more general way to do it in two actions, one for searching and planning the post action , and one for executing the post action right after the deleted plan? So that we could handle more similar situations in same framework.